cli/compose/loader/windows_path.go

```diff
@@ -0,0 +1,70 @@
+package loader
+
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file contains utilities to check for Windows absolute paths on Linux.
+// The code in this file was copied from the Golang filepath package
+// https://github.com/golang/go/blob/1d0e94b1e13d5e8a323a63cd1cc1ef95290c9c36/src/path/filepath/path_windows.go#L12-L65
+
+import "path"
+
+func isSlash(c uint8) bool {
+	return c == '\\' || c == '/'
+}
+
+// isAbs reports whether the path is a Windows absolute path.
+func isAbs(filePath string) (b bool) {
```

Comment:
should we have unit tests for `isAbs`?

Reply:
All these were copied from the golang source code; I did have a quick peek if I could copy their tests as well, but couldn't find a test for these functions in isolation (at a glance)

Reply:
oh; found where they live; I'll add some tests